helix-view/src/editor.rs

```diff
@@ -130,6 +130,10 @@ pub struct Config {
     pub cursorcolumn: bool,
     /// Gutters. Default ["diagnostics", "line-numbers"]
     pub gutters: Vec<GutterType>,
+    /// Minimum number of characters to display as part of line number gutter. Defaults to 2.
+    pub line_numbers_width_min: usize,
+    /// Maximum number of characters to display as part of line number gutter, or 0 for no maximum. Defaults to 0 (no maximum).
+    pub line_numbers_width_max: usize,
```

Comment:
Currently uses options `line-numbers-width-min` and `line-numbers-width-max`. There was a proposal [in this comment](https://github.com/helix-editor/helix/issues/1188#issuecomment-982138408) to add `gutter.<gutter name>` sections to the config file so that these options could be stored as:

```toml
[gutter.line-numbers]
width-min = 1
width-max = 4
```

@archseer How would you like these options exposed? Would you like to propose a config syntax that I should aim for?